packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -33,6 +34,7 @@ const SettingsSidebar = ( { sidebarName } ) => (
 					<PostStatus />
 					<LastRevision />
 					<PostTaxonomies />
+					<PostLink />
```

Comment:
Should this be higher up in the list? If you have lots of taxonomies this could get quite buried down in the sidebar.